lib/tasks/bundlers/generateBundle.js

```diff
@@ -11,8 +11,16 @@ const ReaderCollectionPrioritized = require("@ui5/fs").ReaderCollectionPrioritiz
  * @param {Collection} parameters.dependencies Collection to read dependency files
  * @param {Object} parameters.options Options
  * @param {string} parameters.options.projectName Project name
- * @param {Object} parameters.options.bundleDefintion Module bundle definition
+ * @param {Object} parameters.options.bundleDefinition Module bundle definition
+ * @param {string} parameters.options.bundleDefinition.name The module bundle name
+ * @param {string[]} [parameters.options.bundleDefinition.defaultFileTypes=[".js", ".fragment.xml", ".view.xml", ".properties", ".json"]] List of default file types to be included in the bundle
+ * @param {ModuleBundleDefinitionSection[]} parameters.options.bundleDefinition.sections List of module bundle definition sections
  * @param {Object} parameters.options.bundleOptions Module bundle options
+ * @param {boolean} [parameters.options.bundleOptions.optimize=false] If set to 'true' the module bundle gets minified
+ * @param {boolean} [parameters.options.bundleOptions.decorateBootstrapModule=true] If set to 'false', the module won't be decorated with an optimization marker
+ * @param {boolean} [parameters.options.bundleOptions.addTryCatchRestartWrapper=false] Whether to wrap bootable module bundles with a try/catch to filter out "Restart" errors
+ * @param {boolean} [parameters.options.bundleOptions.usePredefineCalls=false] If set to 'true', sap.ui.predefine is used for UI5 modules
```

Comment:
Different default here? According to the code it seems `false` is correct, see https://github.com/SAP/ui5-builder/blob/bd7aa409be340216a88ceb2607e85d951c9de58a/lib/lbt/bundle/Builder.js#L359

Reply:
see above.